pipelines/tekton/aiedge-e2e/tasks/model-registry-download.yaml

```
@@ -0,0 +1,116 @@
+        jq -r --arg MODEL_NAME "$(params.model-name)" '.items[] | select(.name == $MODEL_NAME).id')
+        
+        export MODEL_VERSION_ID=$(curl -X 'GET' \
+        "$(params.model-registry-hostname)/api/model_registry/v1alpha3/registered_models/$MODEL_ID/versions?" -H 'accept: application/json' | \
+        jq -r --arg MODEL_VERSION "$(params.model-version)" '.items[] | select(.name == $MODEL_VERSION).id')
+        
+        echo -n "MODEL ID is $MODEL_ID"
+        echo -n "MODEL VERSION ID is $MODEL_VERSION_ID"
+
+        if [[ ! -z "$MODEL_ID" ]]; then
+          echo -n "fetching parameters from model-registry"
+
+          export FETCH_MODEL=$(curl --silent -X 'GET' \
+          "$(params.model-registry-hostname)/api/model_registry/v1alpha3/model_versions/$MODEL_VERSION_ID" -H 'accept: application/json' | \
+          jq -r --arg MODEL_PARAM "fetch-model" '.customProperties[$MODEL_PARAM].string_value')
+          echo -n "$FETCH_MODEL" | tee $(results.fetch-model.path)
+
+          export GIT_MODEL_REPO=$(curl --silent -X 'GET' \
+          "$(params.model-registry-hostname)/api/model_registry/v1alpha3/model_versions/$MODEL_VERSION_ID" -H 'accept: application/json' | \
+          jq -r --arg MODEL_PARAM "git-model-repo" '.customProperties[$MODEL_PARAM].string_value')
+          echo -n "$GIT_MODEL_REPO" | tee $(results.git-model-repo.path)
+
+          export GIT_MODEL_REVISION=$(curl --silent -X 'GET' \
+          "$(params.model-registry-hostname)/api/model_registry/v1alpha3/model_versions/$MODEL_VERSION_ID" -H 'accept: application/json' | \
+          jq -r --arg MODEL_PARAM "git-model-revision" '.customProperties[$MODEL_PARAM].string_value')
+          echo -n "$GIT_MODEL_REVISION" | tee $(results.git-model-revision.path)
+
+          export MODEL_DIR=$(curl --silent -X 'GET' \
+          "$(params.model-registry-hostname)/api/model_registry/v1alpha3/model_versions/$MODEL_VERSION_ID" -H 'accept: application/json' | \
+          jq -r --arg MODEL_PARAM "model-dir" '.customProperties[$MODEL_PARAM].string_value')
+          echo -n "$MODEL_DIR" | tee $(results.model-dir.path)
+
+          export MODEL_RELATIVE_PATH=$(curl --silent -X 'GET' \
+          "$(params.model-registry-hostname)/api/model_registry/v1alpha3/model_versions/$MODEL_VERSION_ID" -H 'accept: application/json' | \
+          jq -r --arg MODEL_PARAM "modelRelativePath" '.customProperties[$MODEL_PARAM].string_value')
+          echo -n "$MODEL_RELATIVE_PATH" | tee $(results.modelRelativePath.path)
+
+          export S3-BUCKET-NAME=$(curl --silent -X 'GET' \
+          "$(params.model-registry-hostname)/api/model_registry/v1alpha3/model_versions/$MODEL_VERSION_ID" -H 'accept: application/json' | \
+          jq -r --arg MODEL_PARAM "s3-bucket-name" '.customProperties[$MODEL_PARAM].string_value')
+          echo -n "$S3-BUCKET-NAME" | tee $(results.s3-bucket-name.path)
+        fi
+        else
```

```suggestion
      else
```

^I think this should be at the same level of indentation as the outer if condition, rather than the inner one just above it. In other words, it's odd to see the following two words at the same level of indentation:

```bash
fi
else
```